pagerduty/resource_pagerduty_event_orchestration_path_router_test.go

```diff
@@ -23,6 +24,17 @@ func TestAccPagerDutyEventOrchestrationPathRouter_Basic(t *testing.T) {
 	service := fmt.Sprintf("tf-%s", acctest.RandString(5))
 	orchestration := fmt.Sprintf("tf-orchestration-%s", acctest.RandString(5))
 
+	dynamicRouteToByNameInput := &pagerduty.EventOrchestrationPathDynamicRouteTo{
+		LookupBy: "service_name",
+		Regex:    ".*",
+		Source:   "event.custom_details.pd_service_name",
+	}
+	dynamicRouteToByIDInput := &pagerduty.EventOrchestrationPathDynamicRouteTo{
+		LookupBy: "service_id",
+		Regex:    "ID:(.*)",
+		Source:   "event.custom_details.pd_service_id",
+	}
```

Comment:
**Annotation:** I'm using these variables to 1. build the config and 2. use in the verification functions